dateonly.js

```diff
@@ -71,6 +71,10 @@
     return this.toDate().toDateString();
   };
 
+  DateOnly.prototype.toISOString = function() {
+    return pad(this.year, 4) + '-' + pad(this.month + 1, 2) + '-' + pad(this.date, 2);
```

Comment:
You should just be able to do `return this.toDate().toISOString()` and let the native `Date` object do its thing.

Reply:
Wow, thanks for the quick response!

That was my initial thought as well. But, I end up with the date-shift issue.

Let me examplify:

var dateOnly = new DateOnly(20180325);
console.log(dateOnly.valueOf());
console.log(dateOnly.toString());
console.log(dateOnly.toDate().toISOString());
console.log(dateOnly.toDate().toUTCString());

>>>> output <<<<
20180325
Wed Apr 25 2018
2018-04-24T22:00:00.000Z
Tue, 24 Apr 2018 22:00:00 GMT
>>>> <<<<

The ISOString output converts to UTC time, which for me (being in Stockholm UTC+2) results in the date-part being the day before.

Reply:
Ah yeah, that makes sense. Thanks for the thorough explanation.